app/lib/ui/components/component_list.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:ods_flutter_demo/ui/components/cards/component_card_small.dart';
+import 'package:ods_flutter_demo/ui/components/checkboxes/component_checkboxes.dart';
+import 'package:ods_flutter_demo/ui/components/material/component_material.dart';
+import 'package:ods_flutter_demo/ui/components/radio_buttons/component_radio_buttons.dart';
+import 'package:ods_flutter_demo/ui/components/switches/component_switches.dart';
+import 'package:ods_flutter_demo/ui/components/template/component.dart';
+
+List<OdsComponent> odsComponents = [
+  OdsComponent(
+      "Checkboxes",
+      'assets/il_checkbox.svg',
+      "Checkboxes are used to select or deselect an option. Single checkboxes can be used to determine an individual choice and they can be grouped to allow users to select any combination of items.",
+      [Variant("Checkboxes", "OdsCheckbox", ComponentCheckboxes())]),
+  OdsComponent(
+      "Cards",
+      'assets/il_cards.svg',
+      "Cards are important components that can be used to organise and present a number of different types of related information.",
+      [
+        Variant("Small card", "OdsSmallCard", ComponentCardSmall()),
+        Variant("Vertical image first card", "OdsVerticalImageFirstCard",
+            Text("Coming Soon")),
+        Variant("Vertical header first card", "OdsVerticalHeaderFirstCard",
+            Text("Coming Soon")),
```

Comment:
I think we have to remove this variants cause they don't exist for now in the Flutter library and we have dead links.